src/main/java/edu/ucsb/cs56/ucsbapi/academics/curriculums/v1/classes/Section.java

```diff
@@ -86,19 +86,19 @@ public void setGradingOptionCode(String gradingOptionCode) {
         this.gradingOptionCode = gradingOptionCode;
     }
 
-    public int getEnrolledTotal() {
+    public Integer getEnrolledTotal() {
         return this.enrolledTotal;
     }
 
-    public void setEnrolledTotal(int enrolledTotal) {
+    public void setEnrolledTotal(Integer enrolledTotal) {
         this.enrolledTotal = enrolledTotal;
     }
 
-    public int getMaxEnroll() {
+    public Integer getMaxEnroll() {
         return this.maxEnroll;
     }
 
-    public void setMaxEnroll(int maxEnroll) {
+    public void setMaxEnroll(Integer maxEnroll) {
         this.maxEnroll = maxEnroll;
     }
```

Comment:
Doesn't this shift the NPE to whatever caller is unboxing the (possibly null) integer? I think a better solution might be to check if those integers are null and return `0` if they are.

Reply:
Ok, I added a check for null to both functions.